rescript

```diff
@@ -364,14 +351,13 @@ Please pick a different one using the \`-ws [host:]port\` flag from bsb.`);
     // This could cause problems if source builds (generating js files in the same directory) are supported.
     if (!fileName) return true;
 
-    return !(
-      fileName === ".merlin" ||
-      fileName.endsWith(".js") ||
-      fileName.endsWith(".mjs") ||
-      fileName.endsWith(".cjs") ||
-      fileName.endsWith(genTypeFileExtension) ||
-      watchGenerated.indexOf(fileName) >= 0 ||
-      fileName.endsWith(".swp")
+    return (
+      ((fileName.endsWith(".res") ||
+        fileName.endsWith(".resi") ||
+        fileName.endsWith(".ml") ||
+        fileName.endsWith(".mli")) &&
+        !watchGenerated.includes(fileName)) ||
+      fileName === resConfig
```

Comment:
Q. Doesn't this disable watch when using local js files with binding? Usually vendor stuff